x-pack/plugins/security_solution/server/lib/telemetry/endpoint.mocks.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { SavedObjectsFindResponse } from 'src/core/server';
+import { AgentEventSOAttributes } from './../../../../ingest_manager/common/types/models/agent';
+import {
+  AGENT_SAVED_OBJECT_TYPE,
+  AGENT_EVENT_SAVED_OBJECT_TYPE,
+} from './../../../../ingest_manager/common/constants/agent';
+import { Agent } from '../../../../ingest_manager/common';
+import { ENDPOINT_PACKAGE_CONSTANT } from './fleet_saved_objects';
+
+const testAgentId = 'testAgentId';
+const testConfigId = 'testConfigId';
+
+export const MockOSPlatform = 'somePlatform';
```

Comment:
:red_circle: Robert likes us to put doc-style comments on all exports

Reply:
Added